lib/Authentication/Util/JWT/JWTUtility.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * JWT Utility Class
+ * PHP version 8.0+
+ *
+ * @category Class
+ * @package  CyberSource\Authentication\Util\JWT
+ * @author   CyberSource
+ * @link     https://developer.cybersource.com/
+ */
+
+namespace CyberSource\Authentication\Util\JWT;
+
+use Exception;
+use Firebase\JWT\JWT;
+use Firebase\JWT\JWK;
+
+require_once __DIR__ . '/JWTExceptions.php';
+
+/**
+ * JWT Utility Class
+ *
+ * Provides JWT parsing, verification, and JWK handling functionality.
+ *
+ * @category Class
+ * @package  CyberSource\Authentication\Util\JWT
+ * @author   CyberSource
+ */
+class JWTUtility
+{
+    /**
+     * Supported JWT algorithms
+     */
+    private const SUPPORTED_ALGORITHMS = ['RS256', 'RS384', 'RS512'];
+
+    /**
+     * Maximum depth for JSON decoding
+     */
+    private const JSON_DECODE_DEPTH = 512;
+
+    /**
+     * Error messages constants
+     */
+    private const ERROR_MESSAGES = [
```

Comment:
Remove these error messages because you are not using them in some of the lines below.